web/chromecom.js

```diff
@@ -194,14 +197,10 @@ function requestAccessToLocalFile(fileUrl, overlayManager, callback) {
 
     // Use Chrome's definition of UI language instead of PDF.js's #lang=...,
     // because the shown string should match the UI at chrome://extensions.
-    // These strings are from chrome/app/resources/generated_resources_*.xtb.
-    const i18nFileAccessLabel = PDFJSDev.json(
-      "$ROOT/web/chrome-i18n-allow-access-to-file-urls.json"
-    )[chrome.i18n.getUILanguage?.()];
-
-    if (i18nFileAccessLabel) {
+    const lang = chrome.i18n.getUILanguage?.();
+    if (lang && i18nFileAccessLabels[lang]) {
```

Comment:
Can't we reduce the amount of changes necessary here?
```suggestion
    const i18nFileAccessLabel = i18nFileAccessLabels[chrome.i18n.getUILanguage?.()];
    if (i18nFileAccessLabel) {
```